prqlc/prqlc/tests/test.rs

```diff
@@ -322,6 +323,24 @@ fn format() {
 
     ----- stderr -----
     "###);
+
+    // Single file
```

Comment:
```suggestion
    // FIXME: not good tests, since they don't actually test that the code was formatted...
    // Single file
```

Reply:
Yeah, this would be better if we read the file afterwards.